WordPressLint/src/test/java/org/wordpress/android/lint/MissingNullAnnotationDetectorTest.kt

```diff
@@ -0,0 +1,109 @@
+package org.wordpress.android.lint
+
+import com.android.tools.lint.checks.infrastructure.LintDetectorTest
+import com.android.tools.lint.checks.infrastructure.TestLintTask.lint
+import org.junit.Test
+
+
+class MissingNullAnnotationDetectorTest {
```

Comment:
Suggestion (💡): What if we add additional tests to verify the opposite, that this Lint rule will NOT inform when null annotation is NOT missing? 🤔

Reply:
Good idea 👍 .